docs/howtos/layers/image.md

```diff
@@ -395,19 +395,54 @@ Because the contrast limits are defined by two values, the corresponding slider
 has two handles: one adjusts the smaller value, and one adjusts the larger
 value.
 
-As of right now, adjusting the contrast limits has no effect for `rgb` data.
+For RGB images, adjusting the contrast limits adjusts the black point and white point of the image.
 
+```{important}
+If you pass `contrast_limits` as a keyword argument to a layer, then the full
+extent of the `contrast limits:` range slider will also be set to those values!
+```
+
+```{tip}
 If contrast limits are not passed, napari will compute them. If your data is
 small, napari will take the minimum and maximum values across your
 entire image. If your data is exceptionally large, this operation can be very
-time consuming and so if you pass an image pyramid then napari will use only the
+time consuming. If you pass an image pyramid, then napari will use only the
 top level of that pyramid, or it will use the minimum and maximum values
 across the top, middle, and bottom slices of your image. In general, if working
 with big images, it is recommended to explicitly set the contrast limits if you
 can.
+```
+
+For more precise control, you can right-click on the contrast limits slider to show a version of the slider with numerical values. Importantly, all four displayed values can be edited by clicking on them. Editing the numbers above the circular slider handles allows you to precisely specify values for the two contrast limits. Press the {kbd}`Enter` or {kbd}`Return` to confirm the new value.
+Meanwhile, the numbers at the two ends of the horizontal rule let you change the *range* of the contrast limits slider (programmatically accessed using the `contrast_limits_range` property). Clicking the `full range` button will reset the contrast limits slider range back to the full range of the data type (e.g. 0 and 255 for `uint8`). Finally, clicking the `reset` button will reset *both* the contrast limits *and* the contrast limits range to the full range of the data type.
```

Comment:
```suggestion
Meanwhile, the numbers at the two ends of the horizontal rule let you change the *range* of the contrast limits slider (programmatically accessed using the `contrast_limits_range` property). Clicking the `full range` button will reset the contrast limits slider range back to the full range of the data type (e.g. 0 and 255 for `uint8`). Finally, clicking the `reset` button will reset *both* the contrast limits *and* the contrast limits range to the full range for uint8 images (0, 255), and to the minimum and maximum of the *data* for other data types.
```
Ooof this is very confusing, but correct.
uint8 is a special case where full range is always used.
data range is used for everything else -- and there is no easy way to reset to full range!